packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -744,12 +756,15 @@ class _PopupMenuRoute<T> extends PopupRoute<T> {
     return SafeArea(
       child: Builder(
         builder: (BuildContext context) {
+          final MediaQueryData mediaQuery = MediaQueryData.fromWindow(ui.window);
```

Comment:
The SafeArea here was recently added in #67578, could we remove the SafeArea here since you are accounting for it above and remove this need to call to the window?

Reply:
Nice suggestion 👍

Reply:
Removed the `SafeArea`, change `MediaQueryData.fromWindow` to `MediaQuery.of(context)`, because we need the padding to avoid unavailable areas.